### ISSUE TITLE
make writeProperties() function overwrite any existing run.properties

### ISSUE THREAD

**Issue by jasonfleming** [StormSurgeLive/asgs#426]
If the ASGS stops or is stopped at just the right moment, it may leave behind a stray run.properties file in the run directory. This file should be eliminated by the call to writeProperties so that it is not accidentally appended, potentially creating a run.properties with duplicate or conflicting values.

**Comment by wwlwpd** [comment]
Do you know when that "moment" is?

**Comment by jasonfleming** [comment]
Sort of. The main issue is that the run.properties file for a nowcast (for example) must be written to $RUNDIR (rather than $SCENARIODIR) before the script to download the latest meteorological data is executed. 

Since the beginnings of the run.properties file are written before we actually know what the latest cycle is, we can't write it to $SCENARIODIR (which is named according to the latest cycle number), and if the Operator kills ASGS when it is looking for the next meteorological cycle, and then restarts it later, the old run.properties file will still be in $RUNDIR. 

Basically, the way the scripts currently detect and record the latest meteorological data and handle metadata is a mess and needs to be rethought. However, the rethink must make reliability under all possible scenarios the top priority, even above code elegance, approachability, or maintainability.

**Comment by jasonfleming** [comment]
Not sure when this got fixed but I see the `>` instead of the `>>` operator at the top of `writeProperties.sh` so I think we are all good.
